### ISSUE TITLE
[RFC]: Don't generate file if the template is deleted from custom generator

### ISSUE THREAD

**Issue by simoncrypta** [redwoodjs/redwood#7654]
### Summary

When we set up a custom generator for let say a component, it would be nice if we deleted the test file, it will not generate a test file on a new generated component. 

Right now, if a template is missing from a customer generator folder, it would take the default template.

### Motivation

We already have CLI option to not have test or stories, but it would be more convenient if it can be set by default.

### Detailed proposal

Check if the folder for a custom generator exist and if a template is missing, don't generate this file. 

### Are you interested in working on this?

- [X] I'm interested in working on this

**Comment by jtoar** [comment]
@simoncrypta that seems reasonable to me. @cannikin what do you think?

**Comment by cannikin** [comment]
Hmm, this would be extremely backwards incompatible wouldn't it? If anyone had deleted test/story files, assuming the current behavior (just generate the default template) then they're going to be very surprised when they suddenly don't generate them after this release! Do we need some kind of codemod that puts back any deleted file, to make sure you have a complete set of them?

**Comment by cannikin** [comment]
Yeah I think so—it looks in that custom directory, and only there, to decide what templates to generate. If it doesn't exist, don't create it!
